13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -118,7 +118,7 @@
     "# update the pip package installer\n",
     "!{sys.executable} -m pip install --upgrade --user pip\n",
     "# install required packages\n",
-    "!{sys.executable} -m pip install --upgrade --user uproot awkward vector numpy matplotlib"
+    "!{sys.executable} -m pip install --upgrade --user uproot awkward vector numpy matplotlib aiohttp requests"
```

Comment:
Sorry to jump in, but I think it'd be a great idea. Maybe, given that the needed packages are almost always the same, we could try to create some requirements.txt files for each group of notebooks (e.g. uproot, pyroot, etc.), so that we set up a standard

Reply:
Hi @zlmarshall, indeed this is a problem we're already facing, with some methods deprecated between different versions. Fixing the versions would fix this, at the same time methods are sometimes deprecated due to problems.
I'm happy in any case to add a common setup file along the lines of what @Soap2G mentions, as the initial setup is present for some notebooks but not others, which then poses a problem when running things locally.

Reply:
Awesome, thanks!!